game.py

```diff
@@ -14,13 +14,29 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:   
+        guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if guess in snowman_word:
+            correct_letter_guess_statuses[guess] = True
+        else:
+            wrong_guesses_list += [guess]  
```

Comment:
When adding elements to a list, prefer to use `append`. 

In general case append will add one item to the list, while += will copy all elements of right-hand-side list into the left-hand-side list.

```suggestion
            wrong_guesses_list.append(guess)  
```
Read more about the difference between `append` and concatenating with `+=` https://stackoverflow.com/questions/725782/in-python-what-is-the-difference-between-append-and